### ISSUE TITLE
inferring filenames of cldf-tables from cldf-standards in json file

### ISSUE THREAD

**Issue by HedvigS** [SimonGreenhill/rcldf#11]
Hi Simon, 

I don't know if this is a function that makes sense in rcldf, but I've come to understand that instead of relying on filenames in cldf releases being stable ('values.csv", 'languages.csv") I should instead check the json metadata file and see which table conforms to the cldf-standard I'm after and pull the filename from there instead.

I wrote up a little script that does this in a bit of a clunky way here:
[https://github.com/HedvigS/cookbook/blob/master/recipes/R/make_lang_values_wide.R](https://github.com/HedvigS/cookbook/blob/master/recipes/R/make_lang_values_wide.R)

if this is accomplishable via rcldf that'd be great, but if this isn't really something rcldf is meant to do that's understandable as well.

**Comment by HedvigS** [comment]
I can use rcldf::cldf() on the json file in the cloned copy of glottolog-cldf json, but I was hoping to have a script that doesn't rely on me having a clone but actually fetching it from the internet. Some way of giving it a URL to a json file and having it evaluate the contents. Or something else in that vein, I'm feeling a bit lost.

I also get error messages with rcldf::cldf()

```

> df <- cldf("../../../glottolog-cldf/cldf/cldf-metadata.json")
Column `YEAR` contains character strings.
              No coercion to numeric applied.
Warning messages:                                                                                                                                                                      
1: `as_data_frame()` was deprecated in tibble 2.0.0.
Please use `as_tibble()` instead.
The signature and semantics have changed, see `?as_tibble`.
This warning is displayed once every 8 hours.
Call `lifecycle::last_warnings()` to see where this warning was generated. 
2: In bib2df_tidy(bib, separate_names) : NAs introduced by coercion
> summary(df)
A Cross-Linguistic Data Format (CLDF) dataset:
Name: /Users/skirgard/Dropbox/Git/glottolog-cldf/cldf
JSON: /Users/skirgard/Dropbox/Git/glottolog-cldf/cldf
Type: http://cldf.clld.org/v1.0/terms.rdf#StructureDataset
Tables:
  1/4: codes (5 columns, 26 rows)
  2/4: languages (10 columns, 25439 rows)
  3/4: parameters (4 columns, 6 rows)
  4/4: values (8 columns, 152636 rows)
Sources: 9132
> df$tables
$values
Error in get_extent(x) : 
  lazy-load database '/Library/Frameworks/R.framework/Versions/4.0/Resources/library/utf8/R/utf8.rdb' is corrupt
In addition: Warning message:
In get_extent(x) : internal error -3 in R_decompress1
> df$tables$values
Error in get_extent(x) : 
  lazy-load database '/Library/Frameworks/R.framework/Versions/4.0/Resources/library/utf8/R/utf8.rdb' is corrupt
In addition: Warning messages:
1: In get_extent(x) : restarting interrupted promise evaluation
2: In get_extent(x) : internal error -3 in R_decompress1
```

**Comment by HedvigS** [comment]
I don't fully know why, but I'm able to run rcldf::cldf() without errors now.[ I set up an example here](https://github.com/HedvigS/personal-cldf-cookbook/blob/main/R/make_lang_values_wide_local_clone.R). 

I still get some odd behaviour, for example in the values table "and Oleg Mudraks Etymological dictionary of the Altai" is a parameter for the Language ID "Anna Dybo". That seems to have to do with parsing sources.

**Comment by SamPassmore** [comment]
Is this the same problem as #10 ?

**Comment by HedvigS** [comment]
@SamPassmore yes, the original is. The thing about the alignment getting off kilter for the values-tables I believe is unrelated.

**Comment by HedvigS** [comment]
@SamPassmore well spotted :)

**Comment by SimonGreenhill** [comment]
Fixed in a480028
